web/libs/core/src/blocks/TokenSettingsModal/index.tsx

```diff
@@ -59,6 +71,16 @@ export const TokenSettingsModal = ({
           />
         </Form.Row>
       )}
+      <Form.Actions>
+        {saved && (
+          <div className={styles.success}>
+            <IconCheck /> Saved!
+          </div>
+        )}
+        <Button type="button" onClick={onSubmit} disabled={isPending || saved}>
```

Comment:
can we use `action` in `Form` instead of this inconsistent approach?

Reply:
+1 to this, it'll also make the form more keyboard friendly which we collectively need to take into consideration going forward